admin/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="main.js"></script>
+    <script>
+        window.addEventListener('load', main);
```

Comment:
main is undefined. you can define an empty main function in `main.js` if you want

Reply:
All rigth